test/usr/share/netcheck/test-networkmanagerhelper.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python2
+
+import logging
+import NetworkManager
+import networkmanagerhelper
+
+### This is not an automated test script. This is an ugly dumping ground for bits of code I
+#   have used to test the networkmanagerhelper module.
```

Comment:
I'd move this comment before the imports. Also, I would use the reverse indent instead of the ###.